### PR TITLE
Fix acceptance test errors under RSpec 3.3

### PR DESCRIPTION
**Pull request by derekprior**
When the clearance test suite is run with RSpec 3.3, we get
`uninitialized constant RSpec::Rails::FixtureSupport`. There were some
`FixtureSupport` changes made in RSpec 3.3, but they should not impact
our acceptance tests at all.

When looking at the backtrace, I found gems from Clearance's test group
present when it should solely be gems included in our generated test
app. I switched to executing the acceptance test commands against a
clean bundler environment and everything works again.

### CODE REVIEW

File: spec/acceptance/clearance_installation_spec.rb [hunk 1]
```diff
@@ -69,7 +69,10 @@ def successfully(command, silent = true)
       silencer = ""
     end
 
-    return_value = system("#{command} #{silencer}")
+    return_value = Bundler.with_clean_env do
+      system("#{command} #{silencer}")
+    end
```
Comment (croaky) [inline spec/acceptance/clearance_installation_spec.rb]: Makes sense.

**Comment by croaky** [comment]
Looks good to me.

**Comment by mxie** [comment]
Looks good to me too.

**Comment by derekprior** [comment]
Not waiting for travis. He seems to be napping...

**Pull request merged**